externs/browser/html5.js

```diff
@@ -4286,3 +4315,64 @@ Plugin.prototype.length;
 
 /** @type {string} */
 Plugin.prototype.name;
+
+/**
+ * @see https://w3c.github.io/webcomponents/spec/custom/#custom-elements-api
```

Comment:
That's not the current spec. CE is folded directly into the HTML spec now: https://html.spec.whatwg.org/multipage/scripting.html#custom-elements